.gitignore

```diff
@@ -11,6 +11,7 @@
 
 # Garbage files
 build*/
+cmake_build*/
```

Comment:
@oyvindronningstad just a note: this should not be up-streamed, right? I wonder if we should use commit sauces for those commits that are meant to be taken upstream

Reply:
I think it should be upstreamed. In their getting started guide it tells you to make a cmake_build dir, so it makes sense to have it in .gitignore.

I agree on the sauce tags though. Same scheme as in fw-nrfconnect-zephyr?

Reply:
Agreed on tags like fw-nrfconnect-zephyr

Reply:
Let's make it simpler: just use tags for commits that need to stay down-stream only, the rest can be un-tagged